crates/ruma-common/src/api.rs

```diff
@@ -15,7 +15,7 @@
 use std::{convert::TryInto as _, error::Error as StdError, fmt};
 
 use bytes::BufMut;
-use ruma_identifiers::UserId;
+use ruma_common::UserId;
```

Comment:
Files inside `ruma-common` should import things from `crate::`, not `ruma_common::`. The latter only works due to that `extern crate self as ruma_common;` hack that's there to support macros from `ruma-macros` being called inside this crate.